README.md

```diff
@@ -1,5 +1,89 @@
+* Supports broad ecosystem of AI models from OpenAI and Microsoft to open source LLMs from Hugging Face.
+* Speeds development of custom security, compliance and data custody policy from early evaluation to future scale.
+
+Unliked closed source, vendor-controlled environments for integration private conversations, workflow and generative AI, OpenOps provides an open source, customer-controlled platform for AI exploration with full data data control.
+
+| Demo | Use Case Example |
+|:---|:---|
+| [![Watch the video](https://github-production-user-asset-6210df.s3.amazonaws.com/177788/244097585-1dbe51fa-fb6c-411f-9e18-4e2c99d4c2f2.png)](https://community.mattermost.com/files/k4gdq47njfg6uxuzr5toq5eb4a/public?h=_Lu6LPIGENzL15vfKYSw3AId2yKSGAGySMH9nCRBr24) | **Discussion Summarization** - Within the OpenOps sandbox a private discussion can be analyzed and summarized by an AI bot and backend to help end users more quickly and easily consume information without any data leaving the self-contained system.<br><br>See [usages section](#usage) for more examples below. |
+
+## Why OpenOps?
+
+Everyone’s in a race to deploy generative AI solutions, but need to do so in a responsible and safe way. OpenOps lets you run powerful models in a safe sandbox to establish the right safety protocols before rolling out. Here's an example of an evaluation, implementation and iterative rollout process: 
+
+- **Phase 1:** Set up the OpenOps collaboration sandbox, which is a self-hosted service providing multi-user chat and integration with GenAI. 
+
+- **Phase 2:** Evaluate different GenAI providers, from public SaaS services like OpenAI to fully local open source models, based on your security and privacy requirements.
+
+- **Phase 3:** Invite select early adopters and trust and safety colleagues to explore and evaluate based on their workflows. Observe behavior and record user feedback, and identify issues. Iterate on workflows and usage policies. Consider issues such as data leakage, legal / copyright, privacy, response correctness and appropriateness.
+
+- **Phase 4:** Set and implement policies as availability is incrementally rolled out.
+
+## What does OpenOps include? 
+
+Deploying the OpenOps sandbox includes the following componenents: 
+- 🏰 **Mattermost Server** - Open source, self-hosted alternative to Discord and Slack for strict security environments with playbooks/workflow automation, tools integration, real time 1-1 and group messaging, audio calling and screenshare.  
+- 📙 **PostgreSQL** - Database for storing private data from multi-user, chat collaboration discussions and audit history.    
+- 🤖 **Mattermost AI plugin** - Extension of Mattermost platform for AI bot and generative AI integration. 
+- 🦙 **Open Source, Self-Hosted LLM models** - Models for evaluation and use case development via Hugging Face, including GPT4All (runs on a laptop in 4.2 GB) and Falcon LLM (example of leading scaled self-hosted models). 
```

Comment:
We can point to https://github.com/go-skynet/LocalAI as what we are using for the self host llm models. 

Let's change via Hugging Face to "from Hugging Face and other sources" we don't interact directly with hugging face.